pkg/azuredx/azureauth/auth.go

```diff
@@ -0,0 +1,24 @@
+// Package azureauth implements Azure authorization.
+package azureauth
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+// BearerToken extracts the client's token.
+func BearerToken(req *backend.QueryDataRequest) (accessToken string, err error) {
+	s, ok := req.Headers["Authorization"]
```

Comment:
With our [proposal](https://github.com/grafana/grafana/pull/42422) it can be like
```suggestion
	s, ok := req.Headers["X-ID-Token"]
```